lib-model/lib-model-consume-cli/build.gradle

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2017. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id "java-library"
+}
+
+jar{
+    manifest {
+        attributes('Main-Class':'com.uber.nullaway.libmodel.LibModelInfoExtractor')
+    }
+    from {
+        configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
+    }
```

Comment:
Instead of this why not just use the `shadowJar` plugin?  Like here:

https://github.com/uber/NullAway/blob/1c497a7cff1c4d37a0f51abdf98e0759c0b7539a/jar-infer/jar-infer-cli/build.gradle#L37-L51